mlir/test/Dialect/MIOpen/Tuning/db_load_fail.mlir

```diff
@@ -1,5 +1,9 @@
-// RUN: mlir-miopen-driver -p -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s
+// RUN: mlir-miopen-driver -p -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s --check-prefix=CHECK1
+// RUN: mlir-miopen-driver -p -t f16 -miopen-lowering -miopen-affix-params -debug %s 2>&1 | FileCheck %s --check-prefix=CHECK2
 
-// CHECK: Successfully opened connection to PerfDb
+// CHECK1: Successfully opened connection to PerfDb
 // Perfdb does not have an entry for default configurations
-// CHECK: DB load failed
+// CHECK1: DB load failed
```

Comment:
DB load doesn't consider about the data type as well, only the solverid and performance config. Should remove those redundant one as well.

BTW, can existing solver ids handle fp16 cases?